inc/Module/AutoInstall.pm

```diff
@@ -7,7 +7,7 @@ use ExtUtils::MakeMaker ();
 
```

Comment:
Please exclude all the changes under `inc/` because it's irrelevant for this pull request.